Taskfile.yml

```
@@ -199,14 +199,14 @@ tasks:
     cmds:
       - echo "Starting MCP Inspector at http://localhost:6274"
       - echo "Press Ctrl+C to stop the inspector"
-      - DANGEROUSLY_OMIT_AUTH=true npx --yes @modelcontextprotocol/inspector
+      - npx --yes @modelcontextprotocol/inspector
 
   start-inspector-bg:
     desc: Start MCP Inspector in background
     cmds:
       - |
         echo "Starting MCP Inspector in background..."
-        nohup sh -c 'DANGEROUSLY_OMIT_AUTH=true npx --yes @modelcontextprotocol/inspector' > inspector.log 2>&1 &
+        nohup sh -c 'npx --yes @modelcontextprotocol/inspector' > inspector.log 2>&1 &
         sleep 3
```

With this change, users now need the token to login to the UI. To make it easier for developers, it should print the npx command output to the console as well. Let's also give it extra time to start up.

```suggestion
        nohup npx --yes @modelcontextprotocol/inspector 2>&1 | tee inspector.log &
        sleep 5
```